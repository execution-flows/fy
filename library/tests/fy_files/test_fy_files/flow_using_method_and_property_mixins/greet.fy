method greet using greeting:
    with property greeting

    def -> None:
        print(self._greeting)
