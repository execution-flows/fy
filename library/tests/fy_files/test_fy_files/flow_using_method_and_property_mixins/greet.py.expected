import abc

from fy_files.test_fy_files.flow_using_method_and_property_mixins.abc import With_Greeting_PropertyMixin_ABC


class Greet_UsingGreeting_MethodMixin(
    # Property_mixins
    With_Greeting_PropertyMixin_ABC,
    abc.ABC,
):
    def _greet(self) -> None:
        print(self._greeting)
